docs/docs/introduction/running.md

```diff
@@ -10,13 +10,13 @@ description: How to run Instant OpenHIE
 
 Instant OpenHIE is designed to be easily downloaded and run by anyone on any platform. To get started you will need to first install docker and docker-compose which Instant OpenHIE uses to run all of the necessary services and applications. The following links guide you on how to do this for your platform:
 
-* [Docker engine](https://docs.docker.com/install/)
-* [Docker compose](https://docs.docker.com/compose/install/)
+- [Docker engine](https://docs.docker.com/install/)
+- [Docker compose](https://docs.docker.com/compose/install/)
 
 Once these are installed you can download the latest Instant OpenHIE executable from github. Look for the latest release and download the executable that matches your platform under the assets section of the release:
 
-* [Instant OpenHIE](https://github.com/openhie/instant/releases)
+- [Instant OpenHIE](https://github.com/openhie/instant/releases)
 
-Once the download is complete, move it somewhere memorable. From there you may execute it and you will be presented with a commandline interface. Use the arrow keys to navigate it and choose 'Start Instant OpenHIE'. This will initiate some download that my take some time. Once completed it will then startup the Instant OpenHIE services/application. A web page will be opened with links to the started services/application as well as links to their documentation.
+Once the download is complete, move it somewhere memorable. From there you may execute it and you will be presented with a command line interface. Use the arrow keys to navigate it and choose 'Start Instant OpenHIE'. This will initiate some download that my take some time. Once completed it will then startup the Instant OpenHIE services/application. A web page will be opened with links to the started services/application as well as links to their documentation.
```

Comment:
typo: `This will initiate some download that my take some time`

Reply:
```suggestion
Once the download is complete, move it somewhere memorable. From there you may execute it and you will be presented with a command-line interface. Use the arrow keys to navigate it and choose 'Start Instant OpenHIE'. This will initiate a download that may take some time. Once completed it will then startup the Instant OpenHIE services/application. A web page will be opened with links to the started services/application as well as links to their documentation.
```